lib/block/blockdownload.js

```diff
@@ -136,8 +136,6 @@ class BlockDownload extends EventEmitter {
                 });
             }
         });
-
-        console.log("Block download: block download controller created");
```

Comment:
this log is after instantiating a class that does not depend on client input. Therefore this set up may only fail in a dev environment and this log is not needed in prod